pkg/storaged/content-views.jsx

```diff
@@ -86,13 +86,23 @@ export function set_crypto_options(block, readonly, auto) {
             if (!auto)
                 opts.push("noauto");
         }
+        if (nofail !== null) {
+            extract_option(opts, "nofail");
+            if (nofail)
+                opts.push("nofail");
+        }
+        if (netdev !== null) {
+            extract_option(opts, "_netdev");
+            if (netdev)
+                opts.push("_netdev");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-142402-f4c856ed-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)